src/current/_includes/releases/cloud/2023-06-05.md

```diff
@@ -0,0 +1,19 @@
+## June 05, 2023
+
+<h3> Console changes </h3>
+
+- Organizations that have purchased premium support will now see it included in their [invoices](../cockroachcloud/billing-management.html).
+- Cross-cluster [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) are now limited to {{ site.data.products.dedicated }} clusters with a major version greater than or equal to the major version of the source cluster.
+- The [**Restore jobs**](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) tab of the **Backups page** now shows more information about the restore job, such as the source and destination clusters, the restore type, the backup size, and the job progress.
+
+<h3> Security updates </h3>
+
+- The [Organization Audit Logs API](../cockroachcloud/cloud-org-audit-logs.html), which provides logs of events that occur in a Cloud organization, is now [generally available](../{{site.versions["stable"]}}/cockroachdb-feature-availability.html#feature-availability-phases).
+- `ExternalId` support is now re-enabled in the [Customer-managed Encryption Key (CMEK)](../cockroachcloud/cmek-ops-aws.html) capability on AWS.
+
+<h3> Bug fixes </h3>
+
+- The `status` returned by the [`logexport` Cloud API endpoint](../cockroachcloud/export-logs.html#the-logexport-endpoint) is now determined by the readiness of the underlying logging resources, in addition to the latest log export update job state. Before this change, it was possible for `GET` requests to the `logexport` to report a log export status that conflicted with the latest log export update job state or with the state of the logging infrastructure.
+- Fixed a bug where concurrent [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) could run on the same destination cluster.
```

Comment:
Could this lead to incorrectly-restored data or just consume extra resources?

Reply:
looked into the pr, the description says the cluster could become "unusable" https://github.com/cockroachlabs/managed-service/pull/13194